lib/aggregate/bitfield.rb

```diff
@@ -77,5 +77,9 @@ def to_s
     def <=>(other)
       to_s <=> other.to_s
     end
+
```

Comment:
Can this method be named `empty?`? It seems to ~match the contract for Ruby String.

Reply:
I changed it to `empty` then aliased `nil?` to `empty?` since the bitfield classes treat both as the same. If initialized with a nil value, bitfield defaults to an empty string. Then when storing the value, it stores nil if the string is empty.